topics/ui/controls/progress_bar.md

```diff
@@ -21,112 +21,173 @@ Follow the rules for [progress indicators](progress_indicators.md).
+![](02 determinate_and_ indeterminate.png){width=706}
 
 If an indeterminate process reaches a point where its duration can be determined, switch to a determinate progress bar. For example:
 
-![](progress_bar_indeterminate.png){width=308}
-
-![](progress_bar_determinate.png){width=308}
+![](04 determinate_to_indeterminate.png){width=706}
 
 ### Process name and details
 
 A progress bar can have a process name and process details. For wording, follow the rules for [progress text](progress_text.md).
 
 #### Process name
-
-Place a process name above the progress bar:
-
-![](label_above.png){width=308}
-
-If the vertical space is limited, place the process name with a colon on the left:
-
-![](progress_bar_label_left.png){width=308}
-
-If there are several processes in a group, add a bold header. The header is a noun.
-
-![](several_progresses.png){width=308}
-
-Do **not** show a process name for inline processes:
-
-![](progress_bar_tool_window.png){width=605}
+<table style="none" border="false">
+  <tr>
+    <td width="378">
+      <img src="06 process_name.png" alt="Process name above the bar"/>
+    </td>
+    <td>
+      <p>Place a process name above the progress bar.</p>
+    </td>
+  </tr>
+  <tr>
+    <td width="378">
+      <img src="07 process_name_on_the_left.png" alt="Process name on the left"/>
+    </td>
+    <td>
+      <p>If the vertical space is limited, place the process name with a colon on the left.</p>
+    </td>
+  </tr>
+  <tr>
+     <td width="378">
+         <img src="08 header_for_several_processes.png" alt="Header for several processes"/>
+        </td>
+        <td>
+        <p>If there are several processes in a group, add a bold header. The header is a noun.</p>
+     </td>
+  </tr>
+  <tr>
+     <td width="378">
+         <img src="09 inline_processes.png" alt="Inline process"/>
+        </td>
+        <td>
+        <p>Do not show a process name for inline processes.</p>
+     </td>
+  </tr>
+</table>
 
 #### Process details
-
-Place process details under the progress bar in one line:
-
-![](comment.png){width=308}
-
-The length of the comment is limited by the progress bar:
-
-![](comment_long.png){width=308}
-
-If space is limited, show percentage completed:
-
-![](horizontaly.png){width=342}
+<table style="none" border="false">
+  <tr>
+    <td width="378">
+      <img src="10 process_details.png" alt="Process details under the bar"/>
+    </td>
+    <td>
+      <p>Place process details under the progress bar in one line.</p>
+    </td>
+  </tr>
+  <tr>
+    <td width="378">
+      <img src="11 process_details.png" alt="Comment length limited by progress bar."/>
+    </td>
+    <td>
+      <p>The length of the comment is limited by the progress bar.</p>
+    </td>
+  </tr>
+    <tr>
+     <td width="378">
+         <img src="12 process_details.png" alt="Show percentage completed if space is limited"/>
+        </td>
+        <td>
+        <p>If space is limited, show percentage completed.</p>
```

Comment:
I'm not sure we use it anywhere. Also, showing percent is a duplicating information as the same is expressed with the bar itself. Possibly remove this example?